build.sh

```diff
@@ -318,7 +317,8 @@ if buildAll || hasArg libcudf; then
         BMR_DIR=${RAPIDS_ARTIFACTS_DIR:-"${LIB_BUILD_DIR}"}
         echo "Metrics output dir: [$BMR_DIR]"
         mkdir -p ${BMR_DIR}
-        python ${REPODIR}/cpp/scripts/sort_ninja_log.py ${LIB_BUILD_DIR}/.ninja_log --fmt html --msg "$MSG" > ${BMR_DIR}/ninja_log.html
+        echo "$MSG" > ${BMR_DIR}/build_stats.txt
```

Comment:
Once https://github.com/rapidsai/cudf/pull/12869 and https://github.com/rapidsai/shared-action-workflows/pull/51 are merged, any files that are placed in `${BMR_DIR}` will automatically be uploaded to downloads.rapids.ai.

Since `build_stats.txt` is included in `ninja_log.html`, I don't think it makes sense to also upload it individually.

If an arbitrary temporary file is needed for this message information you could consider using [mktemp](https://www.gnu.org/software/autogen/mktemp.html).